static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -104,15 +141,22 @@ export function DisasterEventMapTile(
   }, [props]);
 
   useEffect(() => {
-    // re-draw map if placeInfo, geoJsonData, or disasterEventData changes
-    if (
-      shouldShowMap &&
-      !_.isEqual(props.disasterEventData, prevDisasterEventData.current)
-    ) {
-      prevDisasterEventData.current = props.disasterEventData;
-      draw(placeInfo, geoJsonData, props.disasterEventData);
+    if (shouldShowMap) {
+      draw(
```

Comment:
we would ideally draw things in order of
polygons -> paths -> points
and avoid redrawing the map unnecessarily

we could do this by calling raise after all the relevant draw functions, or draw only after all geojson data has been fetched. otherwise mouse overs etc would be hard to do.

Reply:
we currently already draw the map only after all geojson data has been fetched.

I don't think raise is needed because svg elements are drawn in the order they appear in the html (so last elements are drawn on top) - I've shifted the layer adding to add polygons, then add paths, then add points.

Reply:
great, missed the shouldShowMap update.